server/py/framework/db/sqldb/db.py

```diff
@@ -5995,6 +5994,77 @@ def list_alerts(
             alerts.append(alert)
         return alerts
 
+    def list_and_delete_project_alerts(
```

Comment:
I would just call it `delete_project_alerts` because you don't really list the alerts but just their IDs which is a legitimate "delete" op response